embuilder.py

```diff
@@ -84,6 +84,10 @@ def build_port(port_name, lib_name, params):
   tasks = sys.argv[2:]
   if 'ALL' in tasks:
     tasks = ['libc', 'libc-mt', 'dlmalloc', 'dlmalloc_threadsafe', 'pthreads', 'libcxx', 'libcxx_noexcept', 'libcxxabi', 'gl', 'binaryen', 'bullet', 'freetype', 'libpng', 'ogg', 'sdl2', 'sdl2-image', 'sdl2-ttf', 'sdl2-net', 'vorbis', 'zlib']
+    if os.environ.get('EMCC_WASM_BACKEND') == '1':
+      skip_tasks = {'libc-mt', 'dlmalloc_threadsafe', 'pthreads'}
+      print('Skipping building of %s, because WebAssembly does not support pthreads.' % ', '.join(skip_tasks))
+      tasks = [x for x in tasks if x not in skip_tasks]
```

Comment:
maybe `filter` would be more pythonic? i'm not sure myself. either is fine with me if no one else cares.